migrations/versions/54e57fb2e1c6_add_contract_source_to_records_and_items.py

```diff
@@ -0,0 +1,26 @@
+"""add contract source to records and items
+
+Revision ID: 54e57fb2e1c6
+Revises: e46b30dd3ff5
+Create Date: 2024-06-26 15:04:24.092549
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '54e57fb2e1c6'
+down_revision = 'e46b30dd3ff5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('records', sa.Column('contract_source', sa.ARRAY, index=True))
```

Comment:
Should we name this `contract_sources` since there can be many?

Reply:
You're right and I'll make that change.